File: bm_json_xam_forms/bm_json_xam_forms/bm_json_xam_forms/Model/Adress.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace bm_json_xam_forms.Model
{
    class Adress
    {
        public string City { get; set; }
        public int ZipCode { get; set; }
        public string Street { get; set; }
        public string Number { get; set; }

        public Adress()
        {
            City = "Berlin";
            ZipCode = 10179;
            Street = "Bahnhofstrasse";
            Number = "10a";
        }
    }
}

File: bm_json_android/app/src/main/java/steffen/de/bm_json_android/Model/Adress.kt
package steffen.de.bm_json_android.Model

import android.R.attr.name



data class Adress(
    var city: String = "Berlin",
    var zipCode: Int = 10179,
    var street: String = "Bahnhofstrasse",
    var number: String = "10a"
){
    override fun toString(): String {
        return String
                .format("%s %s %s %s]",
                        city, zipCode, street, number)
    }
}